app/Http/Requests/Photo/SetPhotosTagsRequest.php

```diff
@@ -17,12 +17,17 @@ class SetPhotosTagsRequest extends BaseApiRequest implements HasPhotos, HasTags
 	use HasTagsTrait;
 	use AuthorizeCanEditPhotosTrait;
 
+	public const OVERRIDE_ATTRIBUTE = 'override';
+
+	public bool $override;
```

Comment:
Same thing here.
```suggestion
	public bool $shallOverride;
```

for example.